activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-jpa/src/main/java/org/activiti/cloud/services/modeling/jpa/ProjectJpaRepository.java

```diff
@@ -68,6 +68,9 @@ default Optional<ProjectEntity> findProjectById(String projectId) {
 
     @Override
     default ProjectEntity createProject(ProjectEntity project) {
+        if(project.getConfiguration() == null || project.getConfiguration().getEnableCandidateStarters() == null) {
```

Comment:
does the or short circuit the condition? Otherwise u will have a null pointer exception on the latter

Reply:
yes, it's a short circuit operator